packages/react-environment/src/types/currency-config.interface.ts

```diff
@@ -0,0 +1,21 @@
+import { ChainId, WebbCurrencyId } from '@webb-dapp/apps/configs';
+import { createElement } from 'react';
+
+export enum CurrencyType {
+  WrappableEvmCurrency,
```

Comment:
Can we document this and give examples? Recall that all non-native Substrate tokens currently are also `ORML` tokens so want to make sure semantically this makes sense.

Reply:
i.e. what are `WrappableEvmCurrency` examples, `ORML`, `BridgeCurrency`.

Reply:
Good idea, actually this should be updated to
```
enum CurrencyType {
     Erc20,
     ORML,
     BridgeErc20,
}
```

with a `type WrappableEvmCurrency = Erc20`.

The intent of the WrappableEvmCurrency is to easily distinguish which currencies can be selected in a 'wrap/unwrap' context with the other edge being a bridge currency.

Reply:
Good idea, actually this should be updated to
```
enum CurrencyType {
     Erc20,
     ORML,
     BridgeErc20,
}
```
with a type `WrappableEvmCurrency = Erc20`.

The intent of the WrappableEvmCurrency is to easily distinguish which currencies can be selected in a 'wrap/unwrap' context with the other edge being a bridge currency.

Reply:
Consider `ERC20, ORML, NATIVE`